### DISCUSSION TITLE
Validation cases for Deployable Composite Structure (DCS)

### DISCUSSION THREAD

**Discussion by capsulecorplab** [ODU-CGA-CubeSat/sealion-mission-architecture#18] [Ideas]
This discussion is for brainstorming validation cases for stakeholder need, 2.3 On-orbit validation DeCS experiment

**Comment by capsulecorplab** [comment]
In identifying a validation case, it may be useful, if not necessary, to identify design constraints.

**Reply by capsulecorplab** [reply]
One suggestion is to use accelerometers, as per @BhagatjiJimesh's suggestion

**Reply by BhagatjiJimesh** [reply]
Deployable Composites Structure (DeCS) experiment consists of an accelerometer for deployment verification and analyzes the in-orbit performance data by capturing structural behavior.

**Reply by capsulecorplab** [reply]
@BhagatjiJimesh what sort of transient waveform should we expect to analyze?

**Comment by BhagatjiJimesh** [comment]
Various Deployable Composites Structure (DeCS) experiment confirmation:
1) Accelerometer data on booms
2) Satellite gyro-rates
3) Burn wire mechanism confirmation.

**Reply by capsulecorplab** [reply]
What is the expected frequency for accelerometer data (e.g., 20Hz, 50Hz)?

^Will be contingent on accelerometer component we down-select
